internal/workload/manager.go

```diff
@@ -238,6 +268,11 @@ func (w *WorkloadManager) ensureWorkloadsFromManifestsAreRunning() error {
 				// Workload is not running - start
 				err = w.workloads.Start(&podWithPath.pod)
 				if err != nil {
+					w.eventsQueue = append(w.eventsQueue, &models.EventInfo{
```

Comment:
Are you planning to add events related to pod `run` failures as well in a separate PR?

Reply:
Sure I can.

Reply:
We can do it in separate PR